root/etc/rc.d/rc.local

```diff
@@ -5,7 +5,13 @@ exportKCL
 
 # setup root password
 if [[ ! -z "$KCL_FDI_ROOTPW" ]]; then
-  echo "root:$KCL_FDI_ROOTPW" | chpasswd >/dev/null
+  # remove possible double and/or single quotes surrounding password string
+  KCL_FDI_ROOTPW=`echo $KCL_FDI_ROOTPW | sed -e "s/^'\(.*\)'$/\1/" -e 's/^"\(.*\)"$/\1/'`
```

Comment:
But this only removes possible *surrounding* single or double-quotes and only in pairs. Isn't that want you wanted?
If we don't remove them, next check will fail when the user puts the quotes around.

As for squash - how do I do that? Not a git power-user, sorry...